specs/binding-kafka.spec/src/main/resources/META-INF/zilla/kafka.idl

```
@@ -269,6 +278,7 @@ scope kafka
             int64 timestamp = 0;        // INIT only
             int32 headersSizeMax = 4;   // INIT only
             int64 producerId = -1;      // INIT only
+            int64 filters = 0;          // INIT only
```

Suggest we default to `-1` instead, as filters mask `0` means no match.